snapshotter/config/config.go

```diff
@@ -54,6 +55,11 @@ type resolver struct {
 	Address string `toml:"address"`
 }
 
+type cache struct {
+	EvictOnConnectionFailure bool `toml:"evict_on_connection_failure" default:"true"`
+	PollConnectionFrequency  int  `toml:"poll_connection_frequency"`
```

Comment:
Does it need a default value?

Reply:
@kzys for this type of poll where we don't need realtime accuracy I might suggest a frequency of 5 minutes arbitrarily. Would you agree or push more frequent heartbeat?